js/battle.js

```diff
@@ -2107,8 +2107,9 @@ var Battle = (function () {
 		// 5 = seeking
 		this.playbackState = 0;
 
-		this.backdropImage = BattleBackdrops[0];
-		this.backdropImage = BattleBackdrops[Math.floor(Math.random() * BattleBackdrops.length)];
+		this.backdropImage = BattleBackdrops.gen6[0];
```

Comment:
Well, just changing the default of this.backdropImage from BattleBackdrops[0] to BattleBackdrops.gen6[0], which would be the appropiate default. Both old and new line is pretty much useless, I guess, but I just kept it.